webui/react/src/components/Markdown.module.scss

```diff
@@ -0,0 +1,754 @@
+.base {
```

Comment:
**chore:** I'm guessing this is sourced from something. We can have a credit link? Also if it changes down the line with improvements we will know where to go take a peek.